benchmarks/benchmarks/ci/esmf_regridder.py

```diff
@@ -84,3 +88,58 @@ def setup(self, type):
 
     def time_perform_regridding(self, type):
         _ = self.regridder(self.src)
+
+
+class TimeLazyRegridding:
+    # Prevent repeat runs between setup() runs - data won't be lazy after 1st.
+    number = 1
+    # Compensate for reduced certainty by increasing number of repeats.
+    repeat = (10, 10, 10.0)
```

Comment:
Why do we have reduced certainty?  Because of the lack of `number` iterations? Is fixed 10 samples sufficient?  It looks like ASV would typically take between 2-20 with the default settings[^1].

[^1]: https://asv.readthedocs.io/en/stable/benchmarks.html#id5

Reply:
As discussed offline with @stephenworsley, below is the latest preferred setting:

```python
    # Compensate for reduced certainty by increasing number of repeats.
    #  (setup() is run between each repeat).
    #  Minimum 5 repeats, run up to 30 repeats / 20 secs whichever comes first.
    benchmark_object.repeat = (5, 30, 20.)
```

Reply:
... and yes @jamesp this is due to the lack of `number` iterations, as you say.